independent-projects/arc/atinject-tck-runner/src/test/java/io/quarkus/arc/tck/Spare.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.arc.tck;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import jakarta.inject.Qualifier;
+
+@Qualifier
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Spare {
```

Comment:
Why do we need this qualifier for AtInject TCKs?

Reply:
See the Weld AtInject TCK runner, it's got the exact same thing. This is because CDI butchered `@Named` and an extra qualifier is required to work around that.

Reply:
Ah right, been a while since I saw AtInject TCKs :)